benchmarks/samples/fabric/fabcar/changeCarOwner.js

```diff
@@ -0,0 +1,67 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+module.exports.info = 'Changing car owner.';
+
+let txIndex = 0;
+let carIndex = 0;
+let carNumber;
+let owners = ['Tomoko', 'Brad', 'Jin Soo', 'Max', 'Adrianna', 'Michel', 'Aarav', 'Pari', 'Valeria', 'Shotaro'];
+let bc, contx;
+
+module.exports.init = async function(blockchain, context, args) {
+    bc = blockchain;
+    contx = context;
+
+    while (carIndex < args.assets) {
+        carIndex++;
+
+        carNumber = carIndex.toString();
+        let carOwner = owners[Math.floor(Math.random() * owners.length)];
+    
+        let myArgs = {
+            chaincodeFunction: 'createCar',
+            chaincodeArguments: [carNumber, 'Toyota', 'Prius', 'Blue', carOwner]
+        };
+    
+        await bc.bcObj.invokeSmartContract(contx, 'fabcar', 'v1', myArgs, 30);
+    }
+
+    return Promise.resolve();
+
+};
+
+module.exports.run = function() {
+    txIndex++;
+    
+    carNumber = txIndex.toString();
+    let newCarOwner = owners[Math.floor(Math.random() * owners.length)];
+
+    let args = {
+        chaincodeFunction: 'changeCarOwner',
+        chaincodeArguments: [carNumber, newCarOwner]
+    };
+
+    if (txIndex === carIndex) {
+        txIndex = 0;
+    }
+
+    return bc.bcObj.querySmartContract(contx, 'fabcar', 'v1', args, 30);
```

Comment:
changing the ownership of an item on the ledger implies that this needs to be smart contract invoke, not a query